src/commands/ls.ts

```diff
@@ -41,6 +41,14 @@ export const lsCommand = (yargs: yargs.Argv) =>
     guard(ls)
   );
 
+
+/** Applies any command-specific formatting to the p0 ls display */
+const displayLsResource = (args: string[], key: string) => 
```

Comment:
🤔 we should avoid anything resource-specific in our base / core code. We should instead separate concerns. I think maybe a better way to do this is to add something like `lsKey` to the `ListerOption` type instead, and generically use that.

Reply:
That makes sense, and I see your point. I did consider adding it to the `ListerOption`, but I was concerned it didn't make sense to have this logic live in the API? Since this is purely a display concern where the data returned from the API is sufficient and the CLI just wants to manipulate it prior to the display, I would have expected it to be in the CLI (and also because we don't want this to apply to the Slack slash commands). Do you think that's a concern? (I'm happy to implement it that way, just looking to better understand the reasoning)